src/Symfony/Bundle/FrameworkBundle/Resources/config/identity_translator.xml

```diff
@@ -10,7 +10,9 @@
         <service id="Symfony\Component\Translation\TranslatorInterface" alias="translator" />
```

Comment:
the default identity translator above should be updated as well.. might even be a public alias to `identity_translator` :thinking: